lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -204,6 +204,25 @@ abstract class EcdhPrivateKey {
   // https://tools.ietf.org/html/rfc6090#appendix-B
   Future<Uint8List> deriveBits(int length, EcdhPublicKey publicKey);
 
+  /// Export the [EcdhPrivateKey] as a [PKCS #8][1] key.
+  ///
+  /// Returns private key as a list of bytes.
```

Comment:
```suggestion
  /// Returns the DER encoding of the _PrivateKeyInfo_ structure specified in [RFC 5208][1] as a list of bytes.
```

Reply:
That matches what we say in:

https://pub.dev/documentation/webcrypto/latest/webcrypto/RsassaPkcs1V15PrivateKey/exportPkcs8Key.html

And it lines up with what we say `importPkcs8Key` requires.

-----

Notably, I think it's nice to know that:
 * It's the _PrivateKeyInfo_ structure -- that way you know what to look at in the RFC.
 * It's DER encoded, and not BER, CER or some other ASN.1 variation, it's always DER.
 * It returns bytes, not something based64 encoded -- this is also indicated by the example that uses `package:pem` to wrap the output.